docs/standard/exceptions/handling-com-interop-exceptions.md

```diff
@@ -22,6 +22,13 @@ Managed and unmanaged code can work together to handle exceptions. If a method t
   
  If a method fails in unmanaged code, an exception can be passed to a managed code segment. The topic [HRESULTS and Exceptions](../../../docs/framework/interop/how-to-map-hresults-and-exceptions.md) contains a table showing how HRESULTS map to runtime exception objects.  
 
+## Longjump Behaviors
+Longjump interop is held to the same constraints as exception handling and is currently only supported on Windows.
```

Comment:
Now that I am thinking about it, we may declare setjmp/longjmp interop as unsupported everywhere, even on Windows. We do not seem to have any tests that it actually works, and there are likely number of cases where it does not work well even on Windows, given the long list of portability issues that setjmp/longjmp comes with.